blockapps-strato/azuredeploy.json

```diff
@@ -0,0 +1,231 @@
+{
+    "resources": [
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "name": "[variables('storageAccount')]",
+            "apiVersion": "2015-05-01-preview",
```

Comment:
I suggest making apiVersion a variable so it becomes easier to move to newer APIs. Also suggest using the latest GA API version 2015-06-15